crates/orderbook/src/arguments.rs

```diff
@@ -1,18 +1,19 @@
-use anyhow::{anyhow, Context, Result};
-use model::app_id::AppId;
 use primitive_types::{H160, U256};
 use reqwest::Url;
 use shared::{
-    arguments::display_option, bad_token::token_owner_finder, fee_subsidy::cow_token::SubsidyTiers,
-    http_client, price_estimation::PriceEstimatorType, rate_limiter::RateLimitingStrategy,
+    arguments::display_option, bad_token::token_owner_finder, http_client,
+    price_estimation::PriceEstimatorType, rate_limiter::RateLimitingStrategy,
 };
-use std::{collections::HashMap, net::SocketAddr, num::NonZeroUsize, time::Duration};
+use std::{net::SocketAddr, num::NonZeroUsize, time::Duration};
 
 #[derive(clap::Parser)]
 pub struct Arguments {
     #[clap(flatten)]
     pub shared: shared::arguments::Arguments,
 
+    #[clap(flatten)]
+    pub order_creation: shared::arguments::OrderQuotingArguments,
```

Comment:
One last naming nit. 😅 
After renaming the struct to `OrderQuotingArguments` it would make sense to also rename the field to `order_quoting` here.

Reply:
Sorry, I was blind.. I should really review it more carefully :(